command-expansion-server/src/packages/ampcs/parser.ts

```diff
@@ -0,0 +1,553 @@
+import { xml2js } from "xml-js";
```

Comment:
This effectively open sources this library. Can we just add it as a new repo to NASA-AMMOS and throw it up on NPM? It is also used in our sequence editor so I don't think we should be duplicating the code like this if we can help it.

Reply:
True, I just wanted something to build temporary and possibly merge to develop so we don't block other parts of the command expansion service. 

We can remove it later with the library once it is on `NASA-AMMOS`

@patkenneally what do you think? Maybe this is something that someone on the team can do in parallel?

Reply:
I am happy to do this. Can do it in 10 minutes. Just say the word :)

Reply:
oh, I thought it was a long process? Not sure if you need @patkenneally approval but I am happy if you do it now.

Reply:
Yeah it should be a very quick process once we have the green light. Copying the project to a new repo in NASA-AMMOS. And then pushing it to the @nasa-jpl NPM org (which I already have access to).